tests/__tests__/sitka.test.ts

```diff
@@ -1,17 +1,97 @@
+import { Dispatch } from "redux"
+import { createSitkaAndStore } from "../sitka-test"
+import { createAppStore, Sitka } from "../../src/sitka"
+
+export class SitkaMock<T = {}> extends Sitka {
+  public registeredModules: T
+ }
+
 describe("Sitka", () => {
 
-  test(`setDispatch dispatch was called`, () => {
+  describe('setDispatch', () => {
+    // Setup
+    const { sitka, store } = createSitkaAndStore()
+    let wasMockDispatched: boolean
+    beforeEach(() => {
+      wasMockDispatched = false
+    })
+
+    test(`setDispatch dispatch was called`, () => {
+      const mockDispatch: Dispatch = <T>(action: T) => {
+        wasMockDispatched = true
```

Comment:
no need to create a global variable. just create a wasMockDispatched in each test so the test